features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -0,0 +1,428 @@
+                            link={
+                                getLink("develop.actions.types.preRegistration.learnMore")
+                            }
+                            showEmptyLink={ false }
+                        >
+                            { t("common:learnMore") }
+                        </DocumentationLink>
+                    </>
+                );
+            default:
+                return "";
+        }
+    };
+
+    /**
+     * This renders the toggle button for action status.
+     */
+    const actionToggle = (): ReactElement => {
+
+        const handleToggle = (e: SyntheticEvent, data: CheckboxProps) => {
+            setIsActive(data.checked);
+
+            if (data.checked) {
+
+                changeActionStatus(actionTypeApiPath, actionInitialValues.id, ActionsConstants.ACTIVATE)
+                    .then(() => {
+                        handleSuccess(ActionsConstants.UPDATE);
+                    })
+                    .catch((error: AxiosError) => {
+                        handleError(error, ActionsConstants.UPDATE);
+                    })
+                    .finally(() => {
+                        mutateActions();
+                    });
+            } else {
+                changeActionStatus(actionTypeApiPath, actionInitialValues.id, ActionsConstants.DEACTIVATE)
+                    .then(() => {
+                        handleSuccess(ActionsConstants.UPDATE);
+                    })
+                    .catch((error: AxiosError) => {
+                        handleError(error, ActionsConstants.UPDATE);
+                    })
+                    .finally(() => {
+                        mutateActions();
+                    });
+            }
+        };
+
+        return !isLoading && !showCreateForm && (
+            <Checkbox
+                label={
+                    isActive
+                        ? t("console:manage.features.actions.status.active")
+                        : t("console:manage.features.actions.status.inactive")
+                }
+                toggle
+                onChange={ handleToggle }
+                checked={ isActive }
+                readOnly={ !hasActionUpdatePermissions }
+                data-componentId={ `${ _componentId }-${ actionTypeApiPath }-enable-toggle` }
+                disabled={ !hasActionUpdatePermissions }
+            />
+
+        );
+    };
+
+    const handleSuccess = (operation: string): void => {
+        dispatch(
+            addAlert({
+                description: t("console:manage.features.actions.notification.success." + operation + ".description"),
+                level: AlertLevels.SUCCESS,
+                message: t("console:manage.features.actions.notification.success." + operation + ".message")
+            })
+        );
+    };
+
+    const handleError = (error: AxiosError, operation: string): void => {
+        if (error.response && error.response.data && error.response.data.description) {
```

Comment:
Use null check here.
```suggestion
        if (error.response?.data?.description) {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847